packages/cli/README.md

```diff
@@ -11,3 +11,581 @@ npm install @stacks/cli
 ## Usage
 
 See [documentation](https://docs.blockstack.org/references/stacks-cli)
+
+### Examples
+Open the terminal to try these basic commands
+
+### List of commands
+To see the usage and options for the command in general  
+```shell script
+$ stx
+``` 
+To see a list of subcommands  
+```shell script
+$ stx help
```

Comment:
I'd vote to omit the leading `$` from cli command examples. This makes it harder to use the copy/paste the commands in to your terminal, as you need to then remove the dollar sign.

Reply:
Added these changes.